migrations/2021-10-28-135931_top-version-columns/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE crates ADD COLUMN highest_version VARCHAR;
+ALTER TABLE crates ADD COLUMN highest_stable_version VARCHAR;
+ALTER TABLE crates ADD COLUMN newest_version VARCHAR;
```

Comment:
basically just to avoid the unnecessary `versions` queries for the endpoints that return a `crate`. it's currently somewhat tightly coupled to the `highest` version fields through our use of the `TopVersions` struct.